.gitlab-ci.yml

```diff
@@ -0,0 +1,64 @@
+stages:
+  - test
+  - build
+
+image:                             parity/rust:nightly
+
+variables:
+  CI_SERVER_NAME:                  "GitLab CI"
+  CARGO_HOME:                      "${CI_PROJECT_DIR}/.cargo"
+
```

Comment:
Not sure how this works here, but do we need to set `BUILD_TARGET`, `BUILD_ARCH`, `CARGO_TARGET` envs here?